src/Nethermind/Nethermind.Evm/Precompiles/Bls/G2MSMPrecompile.cs

```diff
@@ -97,4 +100,28 @@ public long DataGasCost(ReadOnlyMemory<byte> inputData, IReleaseSpec releaseSpec
 
         return (res.EncodeRaw(), true);
     }
+
+    private static bool TryDecodePoint(ReadOnlyMemory<byte> inputData, Span<long> rawPoints, Span<byte> rawScalars, int dest, int i)
```

Comment:
Almost duplicate code, maybe can be done with generics?

Reply:
Did it by passing a function instead